ui/pages/confirmations/components/confirm/info/dapp-swap-comparison-utils.ts

```diff
@@ -0,0 +1,167 @@
+import { Hex } from '@metamask/utils';
+import { Interface, TransactionDescription } from '@ethersproject/abi';
+import { QuoteResponse } from '@metamask/bridge-controller';
+import { addHexPrefix } from 'ethereumjs-util';
+import { getNativeTokenAddress } from '@metamask/assets-controllers';
+
+import { Numeric } from '../../../../../../shared/modules/Numeric';
+
+export const ABI = [
+  {
+    constant: true,
+    inputs: [
+      {
+        name: 'commands',
+        type: 'bytes',
+      },
+      {
+        name: 'inputs',
+        type: 'bytes[]',
+      },
+      {
+        name: 'deadline',
+        type: 'uint256',
+      },
+    ],
+    name: 'execute',
+    type: 'function',
+  },
+  {
+    constant: true,
+    inputs: [
+      {
+        name: 'commands',
+        type: 'bytes',
+      },
+      {
+        name: 'inputs',
+        type: 'bytes[]',
+      },
+    ],
+    name: 'execute',
+    type: 'function',
+  },
+];
+
+const getWordsFromInput = (input: string) => {
+  return input?.slice(2).match(/.{1,64}/gu) ?? [];
```

Comment:
Is Regex the best to parse the bytes? Would it be more standard to use `Uint8Array.fromHex` for example?